app/controllers/application_controller.rb

```diff
@@ -6,4 +6,10 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:invite, keys: %i[email organization_id])
     devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :terms_of_service, own_organization_attributes: [:name]])
   end
+
+  private
+
+  def require_admin
+    redirect_to root_path unless current_user.role == 'admin'
```

Comment:
change role check to `current_user.admin?`

Reply:
+ add some flash message

Reply:
agree with Roman